scripts/bootstrap-aio.yml

```diff
@@ -223,6 +223,30 @@
                     rbd_secret_uuid: "{%raw%}{{ cinder_ceph_client_uuid }}{%endraw%}"
       when: "{{ rpco_deploy_ceph | bool }}"
 
+    # We need to write out a specific log_hosts block so that we can set
+    # the required kernel parameters for the ELK components to startup.
```

Comment:
If you want these set for the log hosts only for all deployments, then the right place is to set it in the group_vars for the appropriate group.